Submodule plugins/vst/external/VST_SDK/VST3_SDK/public.sdk contains modified content
diff --git a/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/vst/utility/stringconvert.h b/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/vst/utility/stringconvert.h
index ff910aa..f15ae78 100644
--- a/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/vst/utility/stringconvert.h
+++ b/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/vst/utility/stringconvert.h
@@ -37,6 +37,7 @@
 #pragma once
 
 #include "pluginterfaces/vst/vsttypes.h"
+#include <cstdint>
 #include <string>
 
 //------------------------------------------------------------------------
